internal/scorecard/helpers/selector.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schelpers
+
+import (
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+const SelectorOpt = "selector"
+const DryRunOpt = "dry-run"
+
+func ValidateSelector(selector string) (labels.Selector, error) {
```

Comment:
Is this strictly necessary, since its just a wrapper to `labels.Parse()`? Could we not just use the parse function directly?

Reply:
yep, I was on the fence with this one as well, my thought in leaving it was that someday there might be a need to do a more descriptive validation than what labels.Parse provides back.

Reply:
I think would be more clear to use the parse direct in this case too.

Reply:
i would vote to call `label.Parse()` directly as well. If there's a need for a wrapper function or different implementation later, adding a function for that at the time would be fine, especially since this is all under an `internal` directory.

Reply:
sounds good, I'll remove it since its not necessary at this point.